examples/cifar10/readme.md

```diff
@@ -52,7 +53,8 @@ parse_cifar10(data_dir_path + "/test_batch.bin",
 
 # Grid Search
 One of the most important hyperparameter in deep learning is learning rate. 
-To get stable and better result, let's try [grid search](https://en.wikipedia.org/wiki/Hyperparameter_optimization#Grid_search)
+To get stable and better result, let's try
+ [grid search](https://en.wikipedia.org/wiki/Hyperparameter_optimization#Grid_search)
```

Comment:
@Randl oh, I thought that you inserted this in this PR. It's just the formatter. Up to you, in this or next PR is okay.